lava/classes/player.py

```diff
@@ -74,26 +75,31 @@ async def check_autoplay(self) -> bool:
 
         :return: True if tracks were added, False otherwise.
         """
-        if not self.autoplay or len(self.queue) >= 5:
-            return False
-        self.bot.logger.info(
-            "Queue is empty, adding recommended track for guild %s...", self.guild
-        )
+        while True:
```

Comment:
This looks like creating a continuous loop for each player. When the number of players increases, this seems to have a certain performance impact. Is there some other solution that doesn't beloved with creating a continuously running loop?